### PR TITLE
CheatSheets: Additions and Corrections Pt.2

### PR DESCRIPTION
**Pull request by mayank7**
IA Page links:

https://duck.co/ia/view/es6_cheat_sheet
https://duck.co/ia/view/emacs_cheat_sheet
https://duck.co/ia/view/emacs_idris_mode_cheat_sheet
https://duck.co/ia/view/elementary_cheat_sheet
https://duck.co/ia/view/ed_cheat_sheet
https://duck.co/ia/view/eclipse_cheat_sheet
https://duck.co/ia/view/drush_cheat_sheet
https://duck.co/ia/view/dockerfile_cheat_sheet
https://duck.co/ia/view/docker_cheat_sheet
https://duck.co/ia/view/default_port_numbers_cheat_sheet
https://duck.co/ia/view/d3js_cheat_sheet
https://duck.co/ia/view/css_cheat_sheet
https://duck.co/ia/view/cryptography_cheat_sheet
https://duck.co/ia/view/cmus_cheat_sheet
https://duck.co/ia/view/aoeii_cheat_sheet
https://duck.co/ia/view/crochet_pattern_cheat_sheet
https://duck.co/ia/view/xwing_alliance_cheat_sheet
https://duck.co/ia/view/xcode_cheat_sheet
https://duck.co/ia/view/wp_cli_cheat_sheet
https://duck.co/ia/view/html_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/cmus.json [hunk 1]
```diff
@@ -105,6 +105,14 @@
                 "key": "y"
             },
             {
+                "val": "Moves marked tracks to the position immediately after the selected track",
+                "key": "p"
+            },
+            {
+                "val": "Moves marked tracks to the position immediately before the selected track",
+                "key": "P"
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/cmus.json]: This should probably be `[Shift] [p]` to make it clear

Comment (mayank7) [inline share/goodie/cheat_sheets/json/cmus.json]: @moollaza I changed P to [Shift] [p]

File: share/goodie/cheat_sheets/json/eclipse.json [hunk 2]
```diff
@@ -364,7 +365,7 @@
                 "val": "Open a resource search dialog to find any class"
             }
         ],
-        "Indentions and Comments": [
+        "indentations and Comments": [
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/eclipse.json]: Should be capitalized: `Indentations`

Comment (mayank7) [inline share/goodie/cheat_sheets/json/eclipse.json]: @moollaza 	Changed indentations to Indentations

**Comment by mayank7** [comment]
@moollaza @mintsoft One question, the HackerEarth leaderboard is not showing my name even if I have 85 PRs merged (zeroclickinfo-goodies). They are displaying people from 32 to 1 PRs, why my name is not there ??

**Comment by mayank7** [comment]
@moollaza @mintsoft @GuiltyDolphin  One question, the HackerEarth leaderboard was updated day before yesterday, it is not showing my name even if I have 85 PRs Merged and 57 PRs Open (zeroclickinfo-goodies). They are displaying people from 32 to 1 PRs, why my name is not there ??

https://www.hackerearth.com/sprints/open-source-india-hacks-2016/leaderboard

You can check my details here to verify my details...

HackerEarth: https://www.hackerearth.com/@zmsrms 
Github: https://github.com/mayank7?tab=contributions&period=monthly 
PR's Merged: 
https://github.com/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3Amayank7+is%3Amerged 
PR's Open: https://github.com/duckduckgo/zeroclickinfo-goodies/pulls/mayank7

**Comment by moollaza** [comment]
@mayank7 AFAIK HackerEarth only awards points for contributions that address and fix open Issues, i.e. the ones labeled with `Priority: High/Med/Low`. I assumed you knew that you when you started contributing

**Comment by mayank7** [comment]
@moollaza But, I have seen names of people who haven't fixed any open issue, only added and updated cheat sheets but had 21 PR shown against their name before the latest update of leaderboard.

**Comment by moollaza** [comment]
@mayank7 that may have been a bug then, whichi it looks like they've corrected? I'm not sure though. I think you'll need to speak with someone from HackerEarth.

/cc @hemanth @swapagarwal do you guys happen to know if the current numbers are correct?

File: share/goodie/cheat_sheets/json/cryptography.json [hunk 3]
```diff
@@ -26,6 +26,9 @@
             "val": "A cryptographic algorithm, i.e. a mathematical function used for encryption and decryption.",
             "key": "Cipher"
         }, {
+            "val": "It is a method of encrypting data piece-by-piece, as opposed to encoding a contiguous chunk of data all at once like a block cipher.",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/cryptography.json]: nit: we should remove `It is`

**Comment by swapagarwal** [comment]
AFAIK, it's just a script that checks for contributions for generating the leaderboard; it would be manually checked!

**Comment by moollaza** [comment]
@mayank7 This LGTM :+1:n

**Comment by hemanth** [comment]
Cool!

**Pull request merged**